docs/whats-new.md

```diff
@@ -2,6 +2,31 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.5 (In development)
+
+### MVT Layer
+
+#### Binary mode
+
+The `binary` prop of `MVTLayer` is set to `true` by default.
+
+#### Direct binary parsing
+
+Binary output is now 2-3X faster for large datasets thanks to parsing directly from PBF to binary, rather than going through GeoJSON as an intermediate representation. Speed comparison on some example data sets (MVT tiles parsed per second):
```

Comment:
```suggestion
Mapbox Vector Tiles parsing throughput is now 2-3x faster, due to MVT tiles being parsed directly into binary attributes rather than GeoJSON, and additional work being performed on worker threads. Speed comparison on some example data sets (MVT tiles parsed per second):
```